### PR TITLE
Include new Black Friday Banner

### PR DESCRIPTION
**Pull request by bordoni**
### 🎫 Ticket

[ET-2241] [TEC-5242]

### 🗒️ Description

As you review this, please make sure you keep in mind this is for a Black Friday.

Still haven't touched the existing tests here, so I am expecting them to break, will fix them tomorrow.

### 🎥 Artifacts 
https://i.bordoni.me/GqsJbYmj

[ET-2241]: https://stellarwp.atlassian.net/browse/ET-2241?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[TEC-5242]: https://stellarwp.atlassian.net/browse/TEC-5242?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dpanta94 — APPROVED** [review]

File: src/admin-views/admin-welcome-message.php [hunk 1]
```diff
@@ -39,6 +39,8 @@ class="tribe-events-admin-title__logo"
 		</ul>
 	</div>
 
+	<?php tribe( \TEC\Common\Admin\Conditional_Content\Black_Friday::class )->render_narrow_banner_html(); ?>
```
Comment (stratease) [inline src/admin-views/admin-welcome-message.php:42]: Can we simplify the FQN?

```suggestion
	<?php tribe( Black_Friday::class )->render_narrow_banner_html(); ?>
```

**Review by stratease — CHANGES_REQUESTED** [review]
Minor tweak.

File: src/admin-views/settings/sidebars/default-sidebar.php [hunk 2]
```diff
@@ -30,20 +31,23 @@
...
+			new Attributes(
+				[
+					'alt'  => '',
+					'role' => 'presentation',
+				]
+			)
 		)
 	)
-);
+	->set_title( new Heading( __( 'Finding and extending your calendar', 'tribe-common' ), 2, new Classes( 'tec-settings-form__sidebar-header' ) ) );
```
Comment (Camwyn) [inline src/admin-views/settings/sidebars/default-sidebar.php:47]: Incorrect text domain
```suggestion
	->set_title( new Heading( __( 'Finding and extending your calendar', 'the-events-calendar' ), 2, new Classes( 'tec-settings-form__sidebar-header' ) ) );
```

File: src/admin-views/settings/sidebars/default-sidebar.php [hunk 3]
```diff
@@ -121,4 +125,6 @@
 		)
 );
 
+$sidebar->add_section( $hero_section );
```
Comment (Camwyn) [inline src/admin-views/settings/sidebars/default-sidebar.php:128]: Already commented on the duplication of `add_section` in TCMN...

File: src/admin-views/settings/sidebars/default-sidebar.php [hunk 4]
```diff
@@ -30,20 +31,23 @@
...
-		tribe_resource_url( 'images/settings_illustration.jpg', false, null, Tribe__Events__Main::instance() ),
-		new Attributes(
-			[
-				'alt'  => '',
-				'role' => 'presentation',
-			]
+
+$hero_section = ( new Settings_Sidebar_Section() );
+$hero_section->header_image(
+		new Image(
```
[the remark this answers is not in the document]
Comment (Camwyn) [reply src/admin-views/settings/sidebars/default-sidebar.php:37]:
```suggestion
	new Image(
```
legit, but might then need to adjust everything below?

File: .github/workflows/zip.yml [hunk 5]
```diff
@@ -53,18 +53,22 @@ jobs:
           ref: ${{ steps.settings.outputs.branch }}
 
       # ------------------------------------------------------------------------------
-      # Setup Node.
```
Comment (Camwyn) [inline .github/workflows/zip.yml:56]: Fixing this is good/cool but:

1. It has nothing to do with the rest of the PR
2. aren't we supposed to be replacing TUT with PUP?

**Review by Camwyn — CHANGES_REQUESTED** [review]

File: .github/workflows/zip.yml (hunk 5 above)
Comment (bordoni) [reply .github/workflows/zip.yml:56]: Was required for something else that was broken.

**Pull request merged**